src/layouts/MainSection.test.js

```diff
@@ -96,4 +96,27 @@ describe('MainSection', () => {
     expect(ranks).toHaveLength(3);
     expect(screen.getByText('Ranking: Deaths')).toBeTruthy();
   });
+
+  it('should display Quotes rules on "Rules" button click and after change category', () => {
+    // given
+    const testScores = [
+      { name: 'Player1', score: 10 },
+      { name: 'Player2', score: 30 },
+      { name: 'Player3', score: 20 },
+    ];
+    jest.spyOn(scores, 'getScores').mockReturnValue(testScores);
+    const testMainSection = renderComponent(MainSection());
+    testMainSection.changeMode(GAME_MODE.QUOTES);
+    const menuButtons = testMainSection.querySelectorAll('.menuButton');
+    userEvent.click(menuButtons[1]);
+    const newMenuButtons = testMainSection.querySelectorAll('.menuButton');
+
+    // when
+    userEvent.click(newMenuButtons[1]);
```

Comment:
Tutaj nie bardzo wiadomo, co się dzieje. Klikamy menuButtons, a za chwilę newMenuButtons. 
Jeśli menuButtons[1] coś oznacza, to przypisałbym temu nazwę, lub komentarz, żeby od razu w kodzie było można sobie wyobrazić, co to robi.

Reply:
Super uwaga! Dzięki Tobie przemyślałem to szybciutko i stwierdziłem, że rzeczywiście mogę to zrobić tak, żeby nazwy klikanych przycisków były jasne! Już poprawione

Reply:
Elegancko :)  👍